package.json

```diff
@@ -324,6 +324,6 @@
     "firebase-admin": "^11.10.0 || ^12.0.0 || ^13.0.0"
   },
   "engines": {
-    "node": ">=14.10.0"
+    "node": ">=18.0.0"
```

Comment:
I'm a little surprised we don't cap the Node version. Why is that? Because the CLI guarantees we can't deploy an unsupported version?